src/fastcs_pandablocks/panda/client_wrapper.py

```diff
@@ -63,7 +64,7 @@ async def introspect(
             for name, block_info in raw_blocks.items()
         }
         formatted_blocks = pformat(blocks, indent=4).replace("\n", "\n    ")
-        logging.debug(f"BLOCKS RECEIVED:\n    {formatted_blocks}")
+        LOGGER.debug(f"BLOCKS RECEIVED:\n    {formatted_blocks}")
```

Comment:
If these are very verbose you could inherit `Tracer` and use `self._log_event()` instead and then it can be enabled in the console with `.enable_tracing()`